src/screens/ConstructionSiteDetailScreen.js

```diff
@@ -0,0 +1,162 @@
+import PropTypes from 'prop-types';
+import React, { useContext } from 'react';
+import { ScrollView, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { colors, consts, normalize } from '../config';
+import {
+  BoldText,
+  Icon,
+  Image,
+  RegularText,
+  SafeAreaViewFlex,
+  Title,
+  TitleContainer,
+  WebViewMap,
+  Wrapper,
+  WrapperWithOrientation
+} from '../components';
+import { arrowLeft, locationIconAnchor, location as locationIcon } from '../icons';
+import { useMatomoTrackScreenView } from '../hooks';
+import { ConstructionSiteContext } from '../ConstructionSiteProvider';
+import { momentFormat } from '../helpers';
+import moment from 'moment';
```

Comment:
moment should be auto linted at the top block with our new eslint/prettier setup, right?

Reply:
same as the other one. fixed it in 026308b2533032adc0542d2e374c7c51e81a27b4